src/test/java/org/gridsuite/study/server/ActuatorHealthTest.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import org.gridsuite.study.server.service.RemoteServicesProperties;
+import org.gridsuite.study.server.utils.TestUtils;
+import org.gridsuite.study.server.utils.WireMockUtils;
+import org.gridsuite.study.server.utils.elasticsearch.DisableElasticsearch;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
+
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
+import static org.junit.Assert.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * @author David Braquart <david.braquart at rte-france.com>
+ */
+
+@RunWith(SpringRunner.class)
+@AutoConfigureMockMvc
+@DisableElasticsearch
+@SpringBootTest
+public class ActuatorHealthTest {
+
+    private WireMockServer wireMockServer;
+    private WireMockUtils wireMockUtils;
+    @Autowired
+    private MockMvc mockMvc;
+    @Autowired
+    private ObjectMapper objectMapper;
+    @Autowired
+    private RemoteServicesProperties remoteServicesProperties;
+
+    @Before
+    public void setup() throws IOException {
+        wireMockServer = new WireMockServer(wireMockConfig().dynamicPort());
+        wireMockUtils = new WireMockUtils(wireMockServer);
+        wireMockServer.start();
+
+        remoteServicesProperties.getServices().forEach(s -> s.setBaseUri(wireMockServer.baseUrl()));
+    }
+
+    private List<String> getOptionalServicesNames() {
+        return remoteServicesProperties.getServices().stream()
+                .filter(RemoteServicesProperties.Service::getOptional)
+                .map(RemoteServicesProperties.Service::getName)
+                .toList();
+    }
+
+    @Test
+    public void testActuatorHealthUp() throws Exception {
```

Comment:
Add a test with only some optional services ?

Reply:
too hard to do (need multiple wiremock server ?)